src/rgw/rgw_lua_request.cc

```diff
@@ -807,6 +809,10 @@ int execute(
   lua_pushcclosure(L, RequestLog, FIVE_UPVALS);
   lua_rawset(L, -3);
 
+  if (!test->run(L, s)){
```

Comment:
please create an std::thread in the Background and run the "run()" function there.
execution rate should be configurable, but could be just 1 per second for now